tests/pytorch/nightly/resnet50-mp.libsonnet

```diff
@@ -13,11 +13,48 @@
 # limitations under the License.
 
 local common = import "common.libsonnet";
+local gpus = import "templates/gpus.libsonnet";
 local timeouts = import "templates/timeouts.libsonnet";
 local tpus = import "templates/tpus.libsonnet";
 local mixins = import "templates/mixins.libsonnet";
+local utils = import "templates/utils.libsonnet";
 
 {
+  local gpu_command_base = |||
+    unset XRT_TPU_CONFIG
+    export GPU_NUM_DEVICES=%(num_gpus)s
+    python3 pytorch/xla/test/test_train_mp_imagenet.py \
+    --model=resnet50 \
+    --batch_size=128 \
+    --log_steps=100 \
+    --num_workers=4 \
+    --num_epochs=2 \
+    --datadir=/datasets/imagenet-mini \
+  |||,
+  local resnet50_gpu = common.PyTorchTest {
+    imageTag: "nightly_3.6_cuda",
+    modelName: "resnet50-mp",
+    volumeMap+: {
+      datasets: common.datasetsVolume
+    },
+    jobSpec+:: {
+      template+: {
+        spec+: {
+          containerMap+: {
+            train+: {
+              resources+: {
+                requests: {
+                  cpu: "13.0",
```

Comment:
I tried just using:
```
cpu: "13.0",
memory: "40Gi",
```

but it did not reflect in thee final yaml

Reply:
Hmmm, that shouldn't happen. Can you file a bug?